### PR TITLE
Creation of a "Contribute" page

### PR DESCRIPTION
**Pull request by patjouk**
This page is composed of the major things volunteers can do to help Django Girls.

Right now, the link is at the bottom of the page. It could be useful to put it in the top menu: I realized that "Django Girls stories" are now in the blog so we could remove the direct link to stories and put a "contribute" one. What do you think?

### CODE REVIEW

**Comment by olasitarska** [comment]
I know I should have included that in the trello card... But could we add translation and Patreon here too? 

Other than that, it looks FANTASTIC. Thank you Lucie! 

Any native speaker want to take a look on language? :)

**Comment by olasitarska** [comment]
Ah, to update this pull request, just commit and push to the same branch again :)

File: templates/contribute.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+{% block content %}
+
+    <div class="container">
+
+        <section id="resources">
+            <div class="row">
+                <div class="col-md-12">
+                    <h2>Contribute</h2>
+                    <p>
+                        There are many ways to contribute to Django Girls: you can organize an event, coach, work on the tutorial and help on Gitter. If you don't have time but want to help, you can always support us on <a href="https://www.patreon.com/djangogirls">Patreon</a>.
```
Comment (williln) [inline templates/contribute.html]: I'd say, "If you'd rather help us financially, you can always support us on..."

File: templates/contribute.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                </div>
+                <div class="col-md-6">
+                    <h3>Coach!</h3>
+                    <div class="cover">
+                        <a href="http://djangogirls.org/events">
+                            <img src="{% static 'img/global/contribute/coach.jpg' %}" alt="A coach helping an attendee." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>You have just heard that an event is happening in your city and went to help? Contact the local organizing team and ask if they need more coaches!</p>
```
Comment (williln) [inline templates/contribute.html]: Change to "Have you heard about an event happening in your city and you want to help?"

**Comment by patjouk** [comment]
I put Patreon in the intro and the translation in "improve the tutorial" section.
I'll change it tomorrow, I need a break :wink:

File: templates/contribute.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+                        <p>You will help a 3-person group to install what they need for the event and guide them through our <a href="http://tutorial.djangogirls.org/">tutorial</a>.</p>
+
+                        <p>Don't forget to check our <a href="http://coach.djangogirls.org/">Coach Manual</a>!</p>
```
Comment (williln) [inline templates/contribute.html]: Coaching Manual (to match the title when you open the GitBook).

File: templates/contribute.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            <div class="row">
+                <div class="col-md-6">
+                    <h3>Improve the tutorial!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/tutorial">
+                            <img src="{% static 'img/global/contribute/tutoedit.jpg' %}" alt="A note book and a pen." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Something in the tutorial is not clear or there is a typo? You can improve the tutorial by making pull request on our <a href="https://github.com/DjangoGirls/tutorial">Github repository</a>.</p>
```
Comment (williln) [inline templates/contribute.html]: "Is something in the tutorial not clear, or is there a typo? You can improve the tutorial by making a pull request to our GitHub repository."

File: templates/contribute.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+                        <p>You can also work on the <a href="https://crowdin.com/project/django-girls-tutorial">translation</a> of the tutorial. If you want to join a team or launch a new translation, <a href="mailto:translations@djangogirls.org">contact us</a>.</p>
```
Comment (williln) [inline templates/contribute.html]: You can work on A translation of the tutorial. "The translation" makes it sound like there is only one.

File: templates/contribute.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+                </div>
+                <div class="col-md-6">
+                    <h3>Work on the website!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/djangogirls">
+                            <img src="{% static 'img/global/contribute/debug.jpg' %}" alt="Computer screen with a console and a webpage." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>There is always something to do to make our website better. You can report bugs and typo, fix those or work on new features.</p>
```
Comment (williln) [inline templates/contribute.html]: "You can report (or fix!) bugs and typos, or work on new features."

File: templates/contribute.html [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+            <div class="row">
+                <div class="col-md-6">
+                    <h3>Help on Gitter!</h3>
+                    <div class="cover">
+                        <a href="https://gitter.im/DjangoGirls/tutorial">
+                            <img src="{% static 'img/global/contribute/gitter.jpg' %}" alt="A web page opened on a computer." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Many people are doing our tutorial outside of a workshop. Sometimes, they get stuck and need help. How about spending a few hours in a week to help them?</p>
```
Comment (williln) [inline templates/contribute.html]: "How about spending a few hours a week helping them?"

File: templates/contribute.html [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+                        <p>This <a href="https://gitter.im/DjangoGirls/tutorial">Gitter room </a> is designed to be a friendly space, especially to beginner. Remember that our <a href="https://djangogirls.org/pages/coc/">Code of Conduct</a> also apply there.</p>
```
Comment (williln) [inline templates/contribute.html]: "... especially for beginners. Remember that our Code of Conduct also applies there."

File: templates/contribute.html [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+                </div>
+                <div class="col-md-6">
+                    <h3>Want to do more?</h3>
+                    <div class="cover">
+                        <a href="https://trello.com/b/q7p6jcfg/django-girls">
+                            <img src="{% static 'img/global/contribute/more.jpg' %}" alt="A pile a tattoo." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>You want to do something and it's not list here? You can look at our <a href="https://trello.com/b/q7p6jcfg/django-girls">Trello board</a> and propose new projects to work on.</p>
```
Comment (williln) [inline templates/contribute.html]: "Want to help out in a way that isn't listed here?"

File: templates/contribute.html [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+                    <h3>Coach!</h3>
+                    <div class="cover">
+                        <a href="http://djangogirls.org/events">
+                            <img src="{% static 'img/global/contribute/coach.jpg' %}" alt="A coach helping an attendee." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>You have just heard that an event is happening in your city and went to help? Contact the local organizing team and ask if they need more coaches!</p>
+
+                        <p>You will help a 3-person group to install what they need for the event and guide them through our <a href="http://tutorial.djangogirls.org/">tutorial</a>.</p>
```
Comment (phalt) [inline templates/contribute.html]: _"You will be helping a group of three people..."_ perhaps?

File: templates/contribute.html [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+                    <h3>Work on the website!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/djangogirls">
+                            <img src="{% static 'img/global/contribute/debug.jpg' %}" alt="Computer screen with a console and a webpage." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>There is always something to do to make our website better. You can report bugs and typo, fix those or work on new features.</p>
+
+                        <p>We are waiting for your <a href="https://github.com/DjangoGirls/djangogirls"> pull requests!</a></p>
```
Comment (phalt) [inline templates/contribute.html]: space inside the anchor tag :)

**Comment by williln** [comment]
This is great work, @patjouk! I made some inline notes, as you can see, but I'm so excited to have this page up.

**Comment by patjouk** [comment]
I added "Support us" and "Translate the tutorial" items. I'll need proofreading of those two!
Thanks again @williln for proofreading! :tulip: 

I tried to add "contribute" in the top global menu but I completely destroyed the perfect line :laughing:. I try to remove "stories" and put "contribute" next to "support us" (cf first comment) and it's working. I don't want to add it to this PR before having your opinion on this.

File: templates/contribute.html [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+                <div class="col-md-6">
+                    <h3>Support us!</h3>
+                    <div class="cover">
+                        <a href="https://www.patreon.com/djangogirls">
+                            <img src="{% static 'img/global/contribute/love.jpg' %}" alt="Pixel heart and Django." />
+                        </a>
+                    </div>
+                    <div class="info">
+
+                        <p>Django Girls is a <a href="http://apps.charitycommission.gov.uk/Showcharity/RegisterOfCharities/CharityFramework.aspx?RegisteredCharityNumber=1163260&SubsidiaryNumber=0">Registered Charity</a> and is volunteers run. If you're an individual or a company and want to support us, you can make a <a href="https://www.patreon.com/djangogirls">monthly donations</a>. For a one time donation, you can send it to our <a href="mailto:hello@djangogirls.org">Paypal account</a>.</p>
```
Comment (williln) [inline templates/contribute.html]: "and is volunteer run." Volunteer should be singular. 

Also, you can "make a monthly donation." Again, "donation" should be singular. 

Edit to "You can send one-time donations to our PayPal account." (Note capitalization of PayPal.)

File: templates/contribute.html [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+                </div>
+                <div class="col-md-6">
+                    <h3>Translate the tutorial!</h3>
+                    <div class="cover">
+                        <a href="https://crowdin.com/project/django-girls-tutorial">
+                            <img src="{% static 'img/global/contribute/translate.jpg' %}" alt="A computer screen saying hello and golden balloons saying yay!." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Our main tutorial is already translated in <a href="http://tutorial.djangogirls.org/">8 languages</a> but we always need help to maintain or start translating in a new language!</p>
```
Comment (williln) [inline templates/contribute.html]: "Our tutorial has already been translated into 8 languages, but we always need helping maintaining those translations. Or you can help us start translating the tutorial into a new language!"

**Comment by williln** [comment]
I think removing "stories" and adding "contribute" next to "support us" is fine, personally!

**Comment by patjouk** [comment]
I added the link in the menu and made some proofreading :rose:

**Comment by patjouk** [comment]
I added a new thing in my PR: The quick search in the admin interface was returning a 404. Now, if you use it, it will search in /admin/core/events as it's not possible for the moment to make it search in all [content](https://django-suit.readthedocs.org/en/0.1.7/configuration.html#search-url).

**Comment by aniav** [comment]
I think it's ok!

**Comment by bmispelon** [comment]
Looks good to me too! :+1: :shipit:

**Pull request merged**